### ISSUE TITLE
Django 2.2.8 -  'sql_server.pyodbc' isn't an available database backend.

### ISSUE THREAD

**Issue by JohnnyZ** [ESSolutions/django-mssql-backend#26]
Hello,

I have tried all of the available examples online to get MSSQL support for Django 2.2. It seems that people have had success. Please let me know if I am making an obvious mistake or if django 2.2 is still lacking support.

Thank you in advance.

Current versions
Windows 10
django==2.2.8
django-mssql-backend==2.5.0
pyodbc==4.0.28

**Django DB Config**
```
DATABASES = {
    'default': {
        'ENGINE': 'sql_server.pyodbc',
        # 'ENGINE': 'sqlsever_ado',
        # 'ENGINE': 'django.db.backends.mssql',
        # 'ENGINE': 'django_pyodbc',
        # 'ENGINE': 'django-pyodbc-azure',
        'NAME': 'db',
        'USER': 'sa',
        'PASSWORD': 'pw',
        'HOST': 'localhost',
        'PORT': '1433',
        'ATOMIC_REQUESTS': True,
        'OPTIONS': {
            'driver': 'ODBC Driver 13 for SQL Server',
        },
    }
}
```

**Error**
django.core.exceptions.ImproperlyConfigured: 'sql_server.pyodbc' isn't an available database backend.
Try using 'django.db.backends.XXX', where XXX is one of:
    'mysql', 'oracle', 'postgresql', 'sqlite3'

**Comment by OskarPersson** [comment]
Ensure that django-mssql-backend has been installed correctly and can be found by Django:

1. See how the test application is setup and used (note the usage of dj_database_url): 
 - https://github.com/ESSolutions/django-mssql-backend/blob/master/.travis.yml
 - https://github.com/ESSolutions/django-mssql-backend/tree/master/testapp

2. List installed packages from Django shell to verify that it can be imported correctly (you might have to temporarily change database engine to built-in backend to get rid of error)

```
$ python manage.py shell
>>> try:
>>>    from pip._internal.operations.freeze import freeze as pip_freeze
>>> except ImportError:  # pip < 10.0
>>>    from pip.operations.freeze import freeze as pip_freeze
>>> pip_freeze()
```

**Comment by JohnnyZ** [comment]
Closing this issue

Django at the current point could in fact see this package, however, I believe there may have been conflicting packages from all of the other attempts/packages I tried earlier.

I ran the commands below to print my packages.
```
$python manage.py shell 

try:
    from pip._internal.operations import freeze
except ImportError:  # pip < 10.0
    from pip.operations import freeze
for package in freeze.freeze():
    print(package)
```

If anyone runs into this issue, I suggest removing similar packages and or starting from a fresh virtual environment. 

Thanks for your response.
